src/plugins/expressions/common/execution/execution_contract.ts

```diff
@@ -17,17 +17,19 @@ import { ExpressionAstExpression } from '../ast';
  * `ExecutionContract` is a wrapper around `Execution` class. It provides the
  * same functionality but does not expose Expressions plugin internals.
  */
-export class ExecutionContract<Input = unknown, Output = unknown, InspectorAdapters = unknown> {
+export class ExecutionContract<
+  Input = unknown,
+  Output = unknown,
+  InspectorAdapters extends Adapters = object
+> {
```

Comment:
`InspectorAdapters` needed to extend `Adapters` to match type and `unknown` no longer conforms to that constraint so use `object` instead.